### PR TITLE
chore: date and timestamp func added

### PR DESCRIPTION
**Pull request by porter473**

### CODE REVIEW

File: src/entities/date/Date.re [hunk 1]
```diff
@@ -7,7 +7,8 @@ let fromJsDate = (d: Js.Date.t): t => d;
 let make = (~value: string, ~format: string): t =>
   MomentUtils.fromString(~date=value, ~format)->MomentUtils.toJsDate;
 
-let toString = (~format: string, ~date: t) => MomentUtils.formatFromJsDate(~format, ~date);
+let toString = (~format: string, ~date: t) =>
```
Comment (porter473) [inline src/entities/date/Date.re:10]: Indenation Change

File: src/entities/date/Date.re [hunk 2]
```diff
@@ -25,7 +26,8 @@ let isAfter = (~first_date: t, ~second_date: t) => {
   );
 };
 
-let getDateTimeAfterElapsedTime = (~elapsed_time: float, ~time_unit: TimeUnit.t, ~initial_date: t) => {
+let getDateTimeAfterElapsedTime =
```
Comment (porter473) [inline src/entities/date/Date.re:29]: Indentation Change

File: src/entities/date/Date.re (hunk 1 above)
Comment (kd4001) [reply src/entities/date/Date.re:10]: Should not be there. Please ping the vscode settings. 
Make sure this setting is active.
"reason_language_server.format_width": 100,

File: src/entities/date/Date.re [hunk 3]
```diff
@@ -42,3 +44,9 @@ let tomorrow = (): t => {
   )
   ->MomentUtils.toJsDate;
 };
+
+let getPreviousDateByDays = (~day: int) => {
+  (Js.Date.now() -. day->float_of_int *. 24.0 *. 60.0 *. 60.0 *. 1000.0)
```
Comment (kd4001) [inline src/entities/date/Date.re:49]: Try using Moment.subtract. You might have to work around duration.

**Review by kd4001 — CHANGES_REQUESTED** [review]

**Review by kd4001 — APPROVED** [review]

**Pull request merged**